velox/dwio/parquet/tests/reader/ParquetReaderBenchmark.cpp

```diff
@@ -136,6 +152,11 @@ class ParquetReaderBenchmark {
     return rowReader;
   }
 
+  // This method is the place where we do the read opeartion.
+  // scanSpec contains the setting of filter and null-filter.
```

Comment:
What do you mean null-filter? The filter "IS NULL"? If that's the case, it's one kind of filters and you don't need to separate them.